crates/sui/src/client_commands.rs

```diff
@@ -989,7 +989,7 @@ impl WalletContext {
         object_id: ObjectID,
     ) -> Result<GetRawObjectDataResponse, anyhow::Error> {
         let client = self.get_client().await?;
-        client.read_api().get_object(object_id).await
+        Ok(client.read_api().get_object(object_id).await?)
```

Comment:
To make sure I understand, this is to convert the error from `thiserror` to `anyhow::Error` right?  This pattern hides that intent a bit, is there another way to do it? (Not a big deal if not).

Reply:
Yes, this is to propagate error to anyhow error, we could do `.map_err(|e| anyhow!(e))`, but I think `Ok(result?)` is the preferred pattern for anyhow.

Reply:
Yeah, I saw it in all the other places as well now, I think this is just me learning about the pattern, so feel free to ignore!